design-proposals/mass_machine_type_transition.md

```diff
@@ -32,26 +41,63 @@ This means each workload must be updated to use a `pc-q35-rhel9.x.x` machine typ
 * Allow user to specify certain VMs for updating machine type automation (e.g. by namespace)
 
 ## Non Goals
-While these additions could be beneficial to cluster-admins who wish to use the mass machine type transition (MMTT), these are not covered in the scope of this design proposal: 
+While these additions are beneficial to cluster-admins who wish to use the
+mass machine type transition (MMTT), these will not be included in the initial implementation:
 * Allowing user to filter VMs by running state: a cluster admin may want to only update VMs that are offline.
 * Allowing user to filter VMs in ways other than by namespace; e.g. with label-selector
+* Subcommands that allow the user to monitor and manage the status of the MMTT job and affected VMs
 
-Also not covered in this design proposal is the method in which users/cluster admins will be informed that they have VMs with unsupported machine types.
+These will eventually be implemented in follow ups once the initial design is implemented.
+
+Additionally, this design proposal does not cover the method in which users/cluster admins
+will be informed that they have VMs with unsupported machine types.
 
 ## User Stories
-As a cluster-admin, I want an automated way to update the machine type of all VMs to be compatible with CentOS Stream X without interrupting my workflow.
+As a cluster-admin, I want an automated way to update the machine type of all
```

Comment:
Should this be limited to CentOS Stream only? IMO this could apply to any OS?

Reply:
+1

Unless I'm missing something we should be able to use this mechanism for any machine type / OS